src/Switches.h

```diff
@@ -94,7 +95,35 @@ class SwitchOnCycleDuration {
 		}
 		g_cycleStartTime = nextCycleStartTime;
 	}
+
+	inline void execute(int ifd, int buffer_size) {
```

Comment:
1. Yes i agree.
2. No, Sockperf does not have something like "NOT_IN_USE(x)" and i don't sure it is necessary. There are many areas like this in the code :
```
class SwitchOff {
public:
	inline void execute() {}
	inline void execute(int) {}
//	inline void execute(unsigned long long) {}
	inline void execute(uint64_t) {}
	inline void execute(int, uint64_t) {}
	inline void execute(TicksTime & _ticks) {}
	inline void execute(struct sockaddr_in *clt_addr, uint64_t seq_num, bool is_warmup) {}
	inline void execute(Message *pMsgRequest, Message * pMsgReply) {}
	inline void execute(Message *pMsgRequest) {}
};
```